pkgs/development/interpreters/python/default.nix

```diff
@@ -69,12 +69,13 @@ with pkgs;
               recursivePthLoader
             ;
           };
+          aliases = self: super: lib.optionalAttrs (config.allowAliases or true) (import ../../../top-level/python-aliases.nix lib super) { };
```

Comment:
```suggestion
          aliases = self: super: lib.optionalAttrs (config.allowAliases or true) (import ../../../top-level/python-aliases.nix lib super);
```
```
attempt to call something which is not a function but a set, at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-2/pkgs/development/interpreters/python/default.nix:72:34
```

Reply:
Should be `lib.optionalAttrs (config.allowAliases or true) (import ../../../top-level/python-aliases.nix lib super);`

Reply:
that's possible as well, was just giving a rough indication that it was failing.